src/Parallel/ArrayCollection/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
```

Comment:
don't you need to add this subdirectory to the parent CMakeLists.txt?

Reply:
It is down below...